test/integration_tests/long/test_model_forecasting.py::ModelTrainTests::test_forecast failed

### Search before asking

- [X] I have searched the EvaDB [issues](https://github.com/georgia-tech-db/eva/issues) and found no similar bug report.


### Bug

[circle ci](https://app.circleci.com/pipelines/github/georgia-tech-db/evadb/5321/workflows/354ed4fc-d637-4122-91ba-b3900512ef86/jobs/38271). We probably should not assert the forecast value since it may change with the package versions.

```
=================================== FAILURES ===================================
________________________ ModelTrainTests.test_forecast _________________________

self = <test.integration_tests.long.test_model_forecasting.ModelTrainTests testMethod=test_forecast>

    @forecast_skip_marker
    def test_forecast(self):
        create_predict_udf = """
            CREATE FUNCTION AirForecast FROM
            (SELECT unique_id, ds, y FROM AirData)
            TYPE Forecasting
            PREDICT 'y';
        """
        execute_query_fetch_all(self.evadb, create_predict_udf)
    
        predict_query = """
            SELECT AirForecast(12) FROM AirData;
        """
        result = execute_query_fetch_all(self.evadb, predict_query)
>       self.assertEqual(int(list(result.frames.iloc[:, -1])[-1]), 459)
E       AssertionError: 484 != 459

test/integration_tests/long/test_model_forecasting.py:81: AssertionError
```

### Environment

_No response_

### Are you willing to submit a PR?

- [X] Yes I'd like to help by submitting a PR!

--- a/test/integration_tests/long/test_model_forecasting.py
+++ b/test/integration_tests/long/test_model_forecasting.py
@@ -78,7 +78,8 @@ class ModelTrainTests(unittest.TestCase):
             SELECT AirForecast(12) FROM AirData;
         """
         result = execute_query_fetch_all(self.evadb, predict_query)
-        self.assertEqual(int(list(result.frames.iloc[:, -1])[-1]), 459)
+        self.assertEqual(len(result), 12)
+        self.assertEqual(result.columns, ["airforecast.y"])
 
     @forecast_skip_marker
     def test_forecast_with_column_rename(self):

